db/event.go

```diff
@@ -88,7 +88,9 @@ type jsEventTask struct {
 // Compiles a JavaScript event function to a jsEventTask object.
 func newJsEventTask(funcSource string) (sgbucket.JSServerTask, error) {
 	eventTask := &jsEventTask{}
-	err := eventTask.Init(funcSource)
+	err := eventTask.InitWithLogging(funcSource,
+		func(s string) { base.Errorf(base.KeyJavascriptLog, "Webhook "+base.UD(s).Redact()) },
```

Comment:
Now we have the format string in here, we can fall back to the typical redaction handling instead of concatenating a pre-tagged string as discussed last week. Same with the other logs in this PR. I.e:

```go
base.Errorf(base.KeyJavascriptLog, "Webhook %s", base.UD(s))
```